_posts/blog/2021-04-21-jshell-accumulo-feature.md

```diff
@@ -0,0 +1,141 @@
+---
+Title: JShell Accumulo Feature
+Author: R. Dane Magbuhos
+Reviewers: Christopher Tubbs, Keith Turner
```

Comment:
I'm not a huge fan of either the Author info (for the same reason Apache projects don't use the `@author` tag in source code, because it's a community-maintained group effort, that all of us are responsible for). However, I recognize that blog posts are somewhat different than source code, since they are typically authored at a point in time, rather than continuously maintained. So, I think it's fine if you want to leave that. Just know that it's optional.

I'm also not a fan of the "Reviewers" tag, either, but for additional reasons. First, it is intended to reflect reviewers of the blog post itself, not the work the blog post is describing. So, right now, it should be blank anyway. Second, I don't think it adds any value. The intent of having this seems to be to try to give credit for contributors who may have made significant contributions as editors in the review process. However, for me personally, it feels like I'm being singled out as responsible in some way for its content, rather than the entire project being responsible for its content. I'm also uncomfortable trying to weigh merit against other individuals like this (it raises the question of how much contributing makes you a reviewer, or how much contributing would make you a co-author).

So, feel free to retain your Author tag if you want, but please remove the Reviewers lines for now, and if you add it back later in response to other reviewers, please leave my name off of it, since I'm not comfortable being called out on the blog posts by name. I really wish we hadn't started this precedent in previous blog posts.

Thanks!

Reply:
Thank you for sharing your perspective on the tags. I agree the blog posts should just inform the users about new features and additions to Accumulo and not about who worked on what. At the time of posting, I was following the format from previous blogs and was unaware about this. I'll remove the author and reviewer tags since they are not needed at all.

Reply:
To be clear, this is just my perspective. Other members of the community may have a different perspective. You are welcome to include them at your discretion (just not with my name in the reviewers).